crates/ott-common/src/discovery/dns.rs

```diff
@@ -50,3 +63,25 @@ impl ServiceDiscoverer for DnsServiceDiscoverer {
         DiscoveryMode::Polling(Duration::from_secs(10))
     }
 }
+#[cfg(test)]
+mod test {
+    use super::*;
+    use serde_json::json;
+    use std::net::Ipv4Addr;
+
+    #[tokio::test]
```

Comment:
`tokio::test` is for async tests

```suggestion
    #[test]
```